src/test/java/de/tum/in/www1/artemis/assessment/ParticipantScoreIntegrationTest.java

```diff
@@ -221,6 +224,34 @@ void getCourseScores_asInstructorOfCourse_shouldReturnCourseScores() throws Exce
         assertThat(scoreOfStudent1.regularPointsAchievable).isEqualTo(20.0);
     }
 
+    @Test
+    @WithMockUser(username = TEST_PREFIX + "instructor1", roles = "INSTRUCTOR")
+    void getCourseScores_asInstructorOfCourseWithGradedPresentations_shouldReturnCourseScores() throws Exception {
+        Exam exam = examRepository.findWithExerciseGroupsAndExercisesById(idOfExam).get();
+
+        GradingScale gradingScale = new GradingScale();
+        gradingScale.setPresentationsNumber(2);
+        gradingScale.setPresentationsWeight(20.0);
+        gradingScale.setCourse(exam.getCourse());
+        gradingScaleRepository.save(gradingScale);
```

Comment:
I would suggest to extract this into a helper method since there are similar lines in another test